<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>869</width>
    <height>608</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>个人通信录管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/PC/Pictures/素材/通讯录.jpg</normaloff>C:/Users/PC/Pictures/素材/通讯录.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow &gt; QWidget { background:rgb(239, 248, 255); }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>781</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>91</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <property name="text">
      <string>全部</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>同学</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>同事</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>朋友</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>亲戚</string>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>570</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #A0A0A0;
border-radius: 10px;      
</string>
    </property>
    <property name="placeholderText">
     <string>请输入您要查询的姓名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>35</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="addAction"/>
    <addaction name="amendAction"/>
    <addaction name="deleteAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>排序方式</string>
    </property>
    <addaction name="namesortSAction"/>
    <addaction name="namesortJAction"/>
    <addaction name="birthsortSAction"/>
    <addaction name="birthsortJAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="birthReminding"/>
    <addaction name="birthInMonth"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addAction">
   <property name="text">
    <string>添加联系人</string>
   </property>
  </action>
  <action name="amendAction">
   <property name="text">
    <string>修改联系人</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除联系人</string>
   </property>
  </action>
  <action name="namesortSAction">
   <property name="text">
    <string>按姓名升序</string>
   </property>
  </action>
  <action name="namesortJAction">
   <property name="text">
    <string>按姓名降序</string>
   </property>
  </action>
  <action name="birthsortSAction">
   <property name="text">
    <string>按出生日期升序</string>
   </property>
  </action>
  <action name="birthsortJAction">
   <property name="text">
    <string>按出生日期降序</string>
   </property>
  </action>
  <action name="birthReminding">
   <property name="text">
    <string>生日提醒</string>
   </property>
  </action>
  <action name="birthInMonth">
   <property name="text">
    <string>统计月份人数</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
